controller/ovn-controller.c

```diff
@@ -2562,6 +2564,18 @@ check_northd_version(struct ovsdb_idl *ovs_idl, struct ovsdb_idl *ovnsb_idl,
     return true;
 }
 
+static bool print_records_callback(uintptr_t arg, const ddlog_record *rec, ssize_t weight) {
+    VLOG_INFO("Printing DDlog records\n");
+    char *record_as_string = ddlog_dump_record(rec);
+    if (record_as_string == NULL) {
+        VLOG_INFO(stderr, "failed to dump record\n");
```

Comment:
The `stderr` here shouldn't be. My compiler complains, I'm not sure why yours doesn't?

```
../controller/ovn-controller.c:2571:9: error: incorrect type in argument 3 (different base types)
../controller/ovn-controller.c:2571:9:    expected char const *format
../controller/ovn-controller.c:2571:9:    got struct _IO_FILE [usertype] *extern [addressable] [toplevel] stderr
```